frontend/components/bc/format/FormatPercent.vue

```diff
@@ -41,42 +42,65 @@ const data = computed(() => {
     return { label, className }
   }
   const percent = props.percent ?? calculatePercent(props.value, props.base)
-  const config = { precision: props.precision ?? 2, fixed: props.fixed ?? 2, addPositiveSign: props.addPositiveSign }
+  const config = {
+    precision: props.precision ?? 2,
+    fixed: props.fixed ?? 2,
+    addPositiveSign: props.addPositiveSign,
+  }
   label = formatPercent(percent, config)
   if (props.comparePercent !== undefined) {
     if (Math.abs(props.comparePercent - percent) <= 0.5) {
       className = 'text-equal'
       leadingIcon = faArrowsLeftRight
       compareResult = 'equal'
-    } else if (percent > props.comparePercent) {
+    }
```

Comment:
I could edit the rules, but this way it is easier to `maintain`, because you can get rid of whole `else (if)` blocks more easily

Refactor time :) 
See: https://medium.com/swlh/return-early-pattern-3d18a41bba8